documentation/FortranForCProgrammers.md

```diff
@@ -0,0 +1,256 @@
+<!--
+Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+-->
+
+Fortran For C Programmers
```

Comment:
I think this is really useful.

It's hard to know how much to include, but I think it would be worth saying more about modules and mentioning USE statements. E.g. it is surprising there are no "fully-qualified" names to reference entities in another module.

Reply:
Can do.